samples/at/aggregation_svc/main.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/transaction-wg/seata-golang/samples/at/aggregation_svc/svc"
 )
 
-var configPath = "/Users/scottlewis/dksl/git/1/seata-golang/samples/at/aggregation_svc/conf/client.yml"
+var configPath = "D:/develop/ability/seata-golang/samples/at/aggregation_svc/conf/client.yml"
```

Comment:
can we change this to an env variable ^?

Reply:
yep,that is a question!